src/routes/routes.tsx

```diff
@@ -13,9 +14,13 @@ export const router = createBrowserRouter([
     element: <p>Product List Generic</p>,
   },
   {
-    path: "/product-list/:category",
+    path: "/products",
     element: <ProductList />,
   },
+  {
+    path: "/products/:id",
+    element: <ProductDescription />,
+  },
   {
     path: "/shopping-cart",
     element: <ProductListCart />,
```

Comment:
The routes should be:

/products   -> `<p>Product List Generic</p>` (Not implemented)
/products/category/:id   ->  <ProductList /> list of products by category
/products/:id  ->  <ProductDescription />

Keep the order